sdk/servicebus/Azure.Messaging.ServiceBus/src/Administration/CreateQueueOptions.cs

```diff
@@ -334,7 +341,8 @@ public bool Equals(CreateQueueOptions other)
                 && string.Equals(_userMetadata, otherOptions._userMetadata, StringComparison.OrdinalIgnoreCase)
                 && (AuthorizationRules != null && otherOptions.AuthorizationRules != null
                     || AuthorizationRules == null && otherOptions.AuthorizationRules == null)
-                && (AuthorizationRules == null || AuthorizationRules.Equals(otherOptions.AuthorizationRules)))
+                && (AuthorizationRules == null || AuthorizationRules.Equals(otherOptions.AuthorizationRules))
+                && MaxMessageSizeInKilobytes.Equals(other.MaxMessageSizeInKilobytes))
```

Comment:
Why the `.Equals` use here rather than just `==` since the type for `MaxMessageSizeInKilobyes` is `long` ?

Reply:
It is `Nullable<long>`

Reply:
To be clear, I've no objection or strong opinion - was just curious.

Reply:
Mostly just following the existing convention. They should be equivalent.